system/infra-monitoring/Chart.yaml

```diff
@@ -41,11 +41,11 @@ dependencies:
     version: 0.1.0
     condition: ipmi_exporter.enabled
 
-  - name: vcenter-exporters
-    alias: vcenter_exporters
-    repository: file://vendor/vcenter-exporters
-    version: 0.1.0
-    condition: vcenter_exporters.enabled
+#  - name: vcenter-exporters
+#    alias: vcenter_exporters
+#    repository: file://vendor/vcenter-exporters
+#    version: 0.1.0
+#    condition: vcenter_exporters.enabled
```

Comment:
Why keep that? This can go away forever so it's decluttered. Git offers version tracking if we want to look up something from the past.